### DISCUSSION TITLE
Installation help in windows

### DISCUSSION THREAD

**Discussion by deekshithbangi** [apache/airflow#34319] [Q&A]
Hi everyone, I am newbie. 
I have gone through the installation documentation of airflow but could install it properly. 
**I'm trying to install the airflow using the pip command along with the constraints mentioned in the airflow installation documentation.**
**currently i have python version 3.10.9** 
Can anyone help me out

**Comment by potiuk** [comment]
Airflow does not work on Windows. You need to use WSL2 if you want to install on Windows.
